kubernetes-manifests/frontend.yaml

```diff
@@ -59,6 +59,12 @@ metadata:
     version: v1
   annotations:
     kardinal.dev.service/dependencies: "checkoutservice:grpc,adservice:grpc,recommendationservice:grpc,productcatalogservice:grpc,cartservice:grpc,shippingservice:grpc"
+    kardinal.dev.service/plugins: |
+      - name: github.com/kurtosis-tech/free-currency-api-plugin
+        type: external
+        servicename: free-currency-api
+        args:
+          API_KEY: fca_live_VKZlykCWEiFcpBHnw74pzd4vLi04q1h9JySbVHDF
```

Comment:
we're okay to add this api key right?

Reply:
it's not mandatory to add it here at least you can use it for testing, but I suggest not adding it.
Actually, we could remove all the Kardinal annotations from here because these manifests are used to deploy the app with kubectl and not with Kardinal.

Reply:
facts